packages/react-sdk/src/components/component-meta.ts

```diff
@@ -24,6 +24,7 @@ export const componentCategories = [
   "media",
   "forms",
   "radix",
+  "xml",
```

Comment:
Hmm do we need to show that component in the components panel at all? If yes we need to think where, custom category for it is probably a bit too much?

Reply:
Now hidden under the flag, we will decide later what to do